python/rascaline-torch/tests/utils/power_spectrum.py

```diff
@@ -0,0 +1,51 @@
+import torch
+from packaging import version
+
+from rascaline.torch import System
+from rascaline.torch.calculators import SphericalExpansion
+from rascaline.torch.utils import PowerSpectrum
+
+
+def system():
+    return System(
+        species=torch.tensor([1, 1, 8, 8]),
+        positions=torch.tensor([[0.0, 0, 0], [0, 0, 1], [0, 0, 2], [0, 0, 3]]),
+        cell=torch.tensor([[10, 0, 0], [0, 10, 0], [0, 0, 10]]),
+    )
+
+
+def calculator():
+    return SphericalExpansion(
+        cutoff=5.0,
+        max_radial=6,
+        max_angular=4,
+        atomic_gaussian_width=0.3,
+        center_atom_weight=1.0,
+        radial_basis={
+            "Gto": {},
+        },
+        cutoff_function={
+            "ShiftedCosine": {"width": 0.5},
+        },
+    )
+
+
+def check_operation(PowerSpectrum):
+    # this only runs basic checks functionality checks, and that the code produces
+    # output with the right type
+
+    descriptor = calculator().compute(system(), gradients=["positions"])
+
+    assert isinstance(descriptor, torch.ScriptObject)
+    if version.parse(torch.__version__) >= version.parse("2.1"):
+        assert descriptor._type().name() == "TensorMap"
```

Comment:
this test is not using the PowerSpectrum class. 

```suggestion
def check_operation(calculator):
    # this only runs basic checks functionality checks, and that the code produces
    # output with the right type

    descriptor = calculator.compute(system(), gradients=["positions"])

    assert isinstance(descriptor, torch.ScriptObject)
    if version.parse(torch.__version__) >= version.parse("2.1"):
        assert descriptor._type().name() == "TensorMap"
```

and rename `def calculator()` to `def spherical_expansion_calculator()`